tests/DynamoDbCompositeModelTest.php

```diff
@@ -752,6 +752,16 @@ public function testFindWithColumns()
         $this->assertEquals($expected, $result->toArray());
     }
 
+    public function testBuilderContainsAllWhereClausesWhenGivenArrayOfConditions()
```

Comment:
This test looks identical as the other one. Since this class inherits the non-composite test case, you don't need this method here.

Reply:
Good catch. Removed.